pages/specification/migration/_index.md

```diff
@@ -3,5 +3,78 @@ title: Migrating from older drafts
 section: docs
 ---
 
-The release notes discuss the changes impacting users and implementers:
+### Introduction
 
+Migrations are like version control for your system, allowing your team to define and share the schema progressions. If you have ever had to figure out when a keyword was introduced and how it behaved, you've faced the problem that migration guides solve.
```

Comment:
I'm not sure I 100% follow the version control analogy (and not every JSON Schema user might necessarily understand version control). What about something more direct, like (total brainstorm, feel free to ignore 😅):

> Just as your schemas evolve over time, the JSON Schema specification does as well. This guide provides useful information to make upgrading your schemas across versions of the specification a breeze.

Or something like that?

Reply:
This sounds like it's conflating the [schema migration pattern](https://en.wikipedia.org/wiki/Schema_migration) where database changes are stored in version control as scripts that modify the database schema. This is not related to that concept and has nothing to do with version control, even as an analogy.

Reply:
Thank you Juan.